dist/index.html

```diff
@@ -1,44 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard by @andym80</title>
     <script defer src="runtime.js"></script>
     <script defer src="main.js"></script>
-</head>
-<div>
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+<header>
     <h1>Leaderboard</h1>
-<div class="all">
-    <div class="table-scores">
-        <div class="board-list">
-            <div class="h2">Recent Scores<span class="h2refresh"></span>
-                <button type="submit" id="refresh">refresh</button>
-            </div>
-            <div class="board-list-item">
-                <div class="scores">
-                    <div class="names">
-                        <div class="points">
+</header>
+
+
+    <section class="all">
+        <section class="table-scores">
+            <div class="board-list">
+                <div class="h2">Recent Scores<span class="h2refresh"></span>
+                    <button type="submit" id="refresh">Refresh</button>
+                </div>
+                <div class="board-list-item">
+                    <div class="scores">
+                        <div class="names">
+                            <div class="points">
+                            </div>
                         </div>
                     </div>
                 </div>
             </div>
-        </div>
-    </div>
-</div>
+        </section>
 
-    <div class="input-players">
-        <div class="input-title">Add your score</div>
-        <ul class="form">
-            <li><input class="names" type="text" placeholder="Enter your name"></li>
-            <li><input class="scores" type="text" placeholder="Enter your score"></li>
+        <section class="input-players">
+            <div class="input-title">Add your score</div>
```

Comment:
- [ ] Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections. You can use a `h1` here instead of a `div`